cmd/observe/observe.go

```diff
@@ -179,6 +179,16 @@ programs attached to endpoints and devices. This includes:
 		"to-label", ofilter,
 		"Show only flows terminating in an endpoint with given labels (e.g. \"key1=value1\", \"reserved:world\")"))
 
+	observerCmd.Flags().Var(filterVar(
+		"from-service", ofilter,
+		"Show all flows originating in the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
```

Comment:
I think that the comment about the default namespace should be repeated for `service` and `to-service` as well.

Reply:
Done: https://github.com/cilium/hubble/pull/88/commits/6f2f7b06a0e1b43227381ee49c6d65bdf2a4edc8